CONTRIBUTING.md

```diff
@@ -44,26 +44,43 @@ $ lerna add truffle-mycmd --scope=@truffle/core
 
 ### Create a new command in `@truffle/core`
 
-Create a new file in `packages/core/lib/commands/`, let's call it `mycmd.js`.
+1. Create a new directory in `packages/core/lib/commands/`, let's call it `truffle-mycmd`.
+2. Create 3 javascript files inside the **truffle-mycmd** directory with the filenames **run.js**, **meta.js** and **index.js**:  
+    * **run.js** contains the entry function after a user calls your command. The entry function has to be anonymous and of type async.
+    * **meta.js** contains information such as command name and command description. 
+    * **index.js** exports both the run module and the meta module. 
 
+#### run.js
 ```shell
-$ cat << EOF > core/lib/commands/mycmd.js
-const command = {
+$ cat << EOF > core/lib/commands/truffle-mycmd/run.js
+
+module.exports = async function (options){
+    const mycmd = require("truffle-mycmd");
+    // TODO: write the run command here, something like:
+    // mycmd(options, done)
+};
+EOF
+```
+#### meta.js
+```shell
+$ cat << EOF > core/lib/commands/truffle-mycmd/meta.js
+module.exports = {
   command: "mycmd",
```

Comment:
I guess you might have to change `mycmd` if you change the name of the command to `truffle-mycmd`. There are some other places in here that it will be necessary as well. On the other hand you could just leave it named `mycmd`.